### DISCUSSION TITLE
Can we pass file data instead of filename without saving the file in server, even in temp storage.

### DISCUSSION THREAD

**Discussion by prudhvi-mallavarapu** [PHPOffice/PhpSpreadsheet#4173] [Q&A] [answered]
Hello,

I have a scenerio, where i getting the .XLSX file url from AWS and want to stream the content to read the data and process without storing in temp storage or any other storage.

Thanks,
Prudhvi.

**Comment by oleibman** [comment, marked as answer]
We do not currently support streaming an input zip file (which is what an xlsx file is under the covers).
